### PR TITLE
Version/3.22

### PR DESCRIPTION
**Pull request by lbudai**

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by kira-syslogng** [comment]
Build FAILURE

File: lib/versioning.h [hunk 1]
```diff
@@ -116,6 +116,7 @@
 #define VERSION_3_19 "syslog-ng 3.19"
 #define VERSION_3_20 "syslog-ng 3.20"
 #define VERSION_3_21 "syslog-ng 3.21"
+#define VERSION_3_21 "syslog-ng 3.22"
```
Comment (MrAnno) [inline lib/versioning.h]: Typo: `VERSION_3_21`

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by MrAnno** [comment]
:+1: Please squash the fixup commit.

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by lbudai** [comment]
after 2 approve, self-merging this

**Pull request merged**